modules/other-integrations/pages/execute-dbt-teradata-transformations-in-airflow-with-cosmos.adoc

```diff
@@ -0,0 +1,248 @@
+= Execute dbt teradata transformation jobs in Apache Airflow using Astronomer Cosmos library
+:experimental:
+:page-author: Satish Chinthanippu
+:page-email: satish.chinthanippu@teradata.com
+:page-revdate: July 15th, 2024
+:description: Execute dbt teradata transformation jobs in Apache Airflow using Astronomer Cosmos library
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, business intelligence, enterprise analytics, airflow, queries, dbt, cosmos, astronomer
+:dir: execute-dbt-teradata-transformations-in-airflow-with-cosmos
+:auxdir: execute-dbt-teradata-transformations-in-airflow-with-cosmos
+
+== Overview
+
+This tutorial demonstrates how to install Apache Airflow on a local machine, configure the workflow to use dbt teradata to run dbt transformations using the astronomer cosmos library, and run it against a Teradata Vantage database. Apache Airflow is a task scheduling tool that is typically used to build data pipelines to process and load data. https://astronomer.github.io/astronomer-cosmos/[Astronomer cosmos] library simplifies orchestrating dbt data transformations in Apache Airflow. Using Cosmos, allows running dbt Core projects as Apache Airflow DAGs and Task Groups with a few lines of code.
+In this example, we will explain how to use astronomer cosmos to run dbt teradata transformations in airflow against Teradata vantage database.
+
+== Prerequisites
+* Access to a Teradata Vantage instance, version 17.10 or higher.
++
+include::ROOT:partial$vantage_clearscape_analytics.adoc[]
+* Python 3.8, 3.9, 3.10 or 3.11 installed.
+* python3-env, python3-pip, pipx installed.
+[source, bash]
+sudo apt install -y python3-venv python3-pip pipx
```

Comment:
Same as above this is only applicable to linux. Mac doesn't need it and Windows is not supported, only through WSL which is linux anyway.

Reply:
Added note for WSL on Windows. Wherever tab is required for three platforms, added tabs and removed tabs at not required places.